<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1052</width>
    <height>615</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>480</y>
      <width>191</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Добавить планету</string>
    </property>
   </widget>
   <widget class="QTableWidget" name="tableWidget">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>20</y>
      <width>661</width>
      <height>261</height>
     </rect>
    </property>
    <property name="rowCount">
     <number>6</number>
    </property>
    <property name="columnCount">
     <number>5</number>
    </property>
    <row/>
    <row/>
    <row/>
    <row/>
    <row/>
    <row/>
    <column>
     <property name="text">
      <string>X</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Y</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Vx</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Vy</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Масса</string>
     </property>
    </column>
    <item row="0" column="0">
     <property name="text">
      <string>0</string>
     </property>
    </item>
    <item row="0" column="1">
     <property name="text">
      <string>0</string>
     </property>
    </item>
    <item row="0" column="2">
     <property name="text">
      <string>0</string>
     </property>
    </item>
    <item row="0" column="3">
     <property name="text">
      <string>0</string>
     </property>
    </item>
    <item row="0" column="4">
     <property name="text">
      <string>1.2166E30</string>
     </property>
    </item>
    <item row="1" column="0">
     <property name="text">
      <string>149500000000</string>
     </property>
    </item>
    <item row="1" column="1">
     <property name="text">
      <string>0</string>
     </property>
    </item>
    <item row="1" column="2">
     <property name="text">
      <string>0</string>
     </property>
    </item>
    <item row="1" column="3">
     <property name="text">
      <string>23297.8704870374</string>
     </property>
    </item>
    <item row="1" column="4">
     <property name="text">
      <string>6.083E24</string>
     </property>
    </item>
    <item row="2" column="0">
     <property name="text">
      <string>299000000000</string>
     </property>
    </item>
    <item row="2" column="1">
     <property name="text">
      <string>0</string>
     </property>
    </item>
    <item row="2" column="2">
     <property name="text">
      <string>0</string>
     </property>
    </item>
    <item row="2" column="3">
     <property name="text">
      <string>16474.0822085901</string>
     </property>
    </item>
    <item row="2" column="4">
     <property name="text">
      <string>1.2166E25</string>
     </property>
    </item>
    <item row="3" column="0">
     <property name="text">
      <string>448500000000</string>
     </property>
    </item>
    <item row="3" column="1">
     <property name="text">
      <string>0</string>
     </property>
    </item>
    <item row="3" column="2">
     <property name="text">
      <string>0</string>
     </property>
    </item>
    <item row="3" column="3">
     <property name="text">
      <string>13451.0317972361</string>
     </property>
    </item>
    <item row="3" column="4">
     <property name="text">
      <string>1.8249E25</string>
     </property>
    </item>
    <item row="4" column="0">
     <property name="text">
      <string>598000000000</string>
     </property>
    </item>
    <item row="4" column="1">
     <property name="text">
      <string>0</string>
     </property>
    </item>
    <item row="4" column="2">
     <property name="text">
      <string>0</string>
     </property>
    </item>
    <item row="4" column="3">
     <property name="text">
      <string>11648.9352435187</string>
     </property>
    </item>
    <item row="4" column="4">
     <property name="text">
      <string>2.4332E25</string>
     </property>
    </item>
    <item row="5" column="0">
     <property name="text">
      <string>747500000000</string>
     </property>
    </item>
    <item row="5" column="1">
     <property name="text">
      <string>0</string>
     </property>
    </item>
    <item row="5" column="2">
     <property name="text">
      <string>0</string>
     </property>
    </item>
    <item row="5" column="3">
     <property name="text">
      <string>10419.1244280004</string>
     </property>
    </item>
    <item row="5" column="4">
     <property name="text">
      <string>3.0415E25</string>
     </property>
    </item>
   </widget>
   <widget class="QPushButton" name="pushButton_2">
    <property name="geometry">
     <rect>
      <x>450</x>
      <y>480</y>
      <width>191</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Удалить планету</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_3">
    <property name="geometry">
     <rect>
      <x>750</x>
      <y>480</y>
      <width>191</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Нарисовать</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="radioButton">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>330</y>
      <width>95</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Эйлера</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="radioButton_2">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>360</y>
      <width>95</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Верле</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="radioButton_3">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>390</y>
      <width>151</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Эйлера-Крамера</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="radioButton_4">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>420</y>
      <width>95</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Бимана</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>300</y>
      <width>55</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Схема:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>450</x>
      <y>300</y>
      <width>121</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Шаг по времени</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>750</x>
      <y>300</y>
      <width>141</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Время моделирования</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit">
    <property name="geometry">
     <rect>
      <x>450</x>
      <y>330</y>
      <width>113</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>3600</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_2">
    <property name="geometry">
     <rect>
      <x>750</x>
      <y>330</y>
      <width>113</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>31536000</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_3">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>70</y>
      <width>113</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>40</y>
      <width>101</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Общая энергия</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>110</y>
      <width>101</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Текущее время</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_4">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>140</y>
      <width>113</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>180</y>
      <width>101</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Центр масс Vx</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_5">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>210</y>
      <width>113</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_7">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>250</y>
      <width>101</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Центр масс Vy</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_6">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>280</y>
      <width>113</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1052</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
